README.md

```diff
@@ -2,727 +2,369 @@
+
+- Mbed OS platform name.
+- Mount point (MSD or disk).
+- Serial port.
 
 # Installation
 
 ## Installation from PyPI (Python Package Index)
 
-mbed-ls module is redistributed via PyPI. We recommend you use the [application pip](https://pip.pypa.io/en/latest/installing.html#install-pip).
+To install `mbed-ls` from [PyPI](https://pypi.python.org/pypi/mbed-ls), run the following command:
 
-**Note:** Python 2.7.9 onwards include ```pip``` by default, so you may have ```pip``` already.
-
-To install mbed-ls from [PyPI](https://pypi.python.org/pypi/mbed-ls) use command:
-```
-$ pip install mbed-ls --upgrade
+```bash
+pip install mbed-ls --upgrade
 ```
 
 ## Installation from Python sources
 
-**Prerequisites:** you need to have [Python 2.7.x](https://www.python.org/download/releases/2.7/) or [Python 3.6.x](https://www.python.org/downloads/release/python-362/) installed on your system.
-
-**Note:** if your OS is Windows, please follow the installation instructions [for the serial port driver](https://developer.mbed.org/handbook/Windows-serial-configuration).
+**Prerequisites:** You need to have [Python 2.7.x](https://www.python.org/download/releases/2.7/) or [Python 3.6.x](https://www.python.org/downloads/release/python-362/) installed on your system.
 
-To install the mbed-ls module:
+**Note:** If your OS is Windows, please follow the installation instructions [for the serial port driver](https://os.mbed.com/docs/latest/tutorials/windows-serial-driver.html).
 
-Clone the mbed-ls repository. The following example uses the GitHub command line tools, but you can do this directly from the website:
+To install the `mbed-ls` module, first clone the `mbed-ls` repository. The following example uses the GitHub command-line tools, but you can do this directly from the website:
 
-```
-$ git clone https://github.com/ARMmbed/mbed-ls.git
+```bash
+git clone https://github.com/ARMmbed/mbed-ls.git
+cd mbed-ls
+python setup.py install
 ```
 
-Change the directory to the mbed-ls repository directory:
+# Use
 
-```
-$ cd mbed-ls
+## Command-line
+
+The command-line tool is available with the command `mbedls`:
+
+```bash
+$ mbedls
++---------------+----------------------+-------------+-------------+--------------------------------------------------+-----------------+
+| platform_name | platform_name_unique | mount_point | serial_port | target_id                                        | daplink_version |
++---------------+----------------------+-------------+-------------+--------------------------------------------------+-----------------+
+| K64F          | K64F[0]              | D:          | COM18       | 0240000032044e4500257009997b00386781000097969900 | 0244            |
++---------------+----------------------+-------------+-------------+--------------------------------------------------+-----------------+
 ```
 
-Now you are ready to install mbed-ls.
+### Result formats
+
+The command-line is able to list the results in a number of formats.
```

Comment:
Query: Would it be accurate to say "list the results in many formats"? If so, please change. If not, please delete "a number of".

Reply:
Yes. I would write this as: The Mbed LS command line accepts a few arguments to change the format of the results. The default format is a table. You may pass `--simple` to simplify this table format, and `--json` to print the table as a json list of the rows.